geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractSetEXIntegrationTest.java

```diff
@@ -55,13 +55,6 @@ public void errors_givenWrongNumberOfArguments() {
     assertExactNumberOfArgs(jedis, Protocol.Command.SETEX, 3);
   }
 
-  @Test
-  public void givenMoreThanFourArgumentsProvided_returnsWrongNumberOfArgumentsError() {
-    assertThatThrownBy(
-        () -> jedis.sendCommand("key", Protocol.Command.SETEX, "key", "10", "value", "extraArg"))
-            .hasMessageContaining("ERR wrong number of arguments for 'setex' command");
-  }
-
```

Comment:
Sorry, did a force push after @DonalEvans' comment and lost his comment.  Next time will add a commit instead of rebasing. He pointed out that this test method is redundant as the assertExactNumberOfArgs in the other test already covers this case. This was a test method that was already present in AbstractSetExIntegrationTest, so removed it. I also had copied this test into AbstractPSetExIntegrationTest, but removed it after Donal's comment.